public/.well-known/openid-credential-issuer

```diff
@@ -1,76 +1,84 @@
 {
-    "credential_issuer": "https://credential-issuer.example.com",
-    "authorization_servers": [ "https://server.example.com" ],
-    "credential_endpoint": "https://credential-issuer.example.com",
-    "batch_credential_endpoint": "https://credential-issuer.example.com/batch_credential",
-    "deferred_credential_endpoint": "https://credential-issuer.example.com/deferred_credential",
-    "display": [
-        {
-            "name": "Example University",
-            "locale": "en-US"
-        },
-        {
-            "name": "Example Université",
-            "locale": "fr-FR"
-        }
+    "authorization_endpoint": "https://authz-server1.zenswarm.forkbomb.eu/authorize",
```

Comment:
⚠️ pay attention that the endpoints defined here are the one, that the wallet istance will call for real, by now this is now `${server}/authz_server/authorize`. Be sure to fill it correctly or to move the contracts in the correct place